lib/tools/mp4-inspector.js

```diff
@@ -123,6 +123,30 @@ var
         avgBitrate: view.getUint32(8)
       };
     },
+    edts: function edts(data) {
+      return {
+        boxes: inspectMp4(data)
+      };
+    },
+    elst: function elst(data) {
+      var
+        view = new DataView(data.buffer, data.byteOffset, data.byteLength),
+        result = {
+          version: view.getUint8(0),
+          flags: new Uint8Array(data.subarray(1, 4)),
+          edits: []
+        },
+        entryCount = view.getUint32(4),
+        i;
+      for (i = 8; entryCount; i += 12, entryCount--) {
+        result.edits.push({
+          segmentDuration: view.getUint32(i),
```

Comment:
looking at the mp4 spec it seems like we need to look at `result.version` and if `result.version` is `0` then `segmentDuration` and `mediaTime` will 32 bit and if `result.version` is `1` then segmentDuration` and `mediaTime`  will be 64 bit.

Reply:
for an example of grabbing 64 bit values (i know js doesn't actually support that many bits for numbers, but we can still get them up to 52 bits) see https://github.com/videojs/mux.js/blob/main/lib/tools/parse-sidx.js#L20-L21

Reply:
What about BigInt? Ive used `.getBigUint64`.

Reply:
What is the minimum supported node version for this library?

Reply:
IE11 😂 (sorry)

Reply:
Check it now

Reply:
Ive added 64 bit test case. 
should be all good now